src/mars.d

```diff
@@ -169,6 +169,7 @@ Where:
   -transition=?    list all language changes
   -unittest        compile in unit tests
   -v               verbose
+  -vcg-ast         generates .d.cg file with the ast right before obj-file generation
```

Comment:
Please remove from the help menu, it's an internal tool and I wouldn't want to commit to it too much.